core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -292,6 +292,7 @@ private[spark] object UIUtils extends Logging {
     <html>
       <head>
         {commonHeaderNodes(request)}
+        <script>setAppBasePath('{activeTab.basePath}')</script>
```

Comment:
I think we can move this line to function `commonHeaderNodes`

Reply:
At first, I considered the way you mention but to do so, we need to have an additional parameter in `commonHeaderNodes` to pass `activeTab` only for this purpose.
Which do you think is better?

Reply:
I see. Let's keep it this way. The method `basicSparkPage` also calls the function `commonHeaderNodes` but there is no `activeTab` in it.